t04_refactored.py

```diff
@@ -231,9 +231,36 @@ def team_19_adv():
 ###################################################################################
 
 def team_20_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1aO8daq4WLqOsChlhvmClBHNZEx4faJcmsvGXPyOU990/edit?usp=sharing
+    :return:
+    America Gaona Borges
+    Jessica Douthit
+    """
+    global isDead
+    input("Go with him further Up, or Down? ")
+    if input == "Up":
+        print("You've reached a large vault... a large console stands to its right...")
+        print("The console lights up when you reach it... it asks for a passcode")
+        input("Guess the code" or "Ask for help")
+        if "Ask for help":
+            print("You ask Dr. Heggen for help")
+            print("He pulls out a small electrical device and plugs it into the console")
+            print("The console glows green and the vault opens up")
+            print("you've found riches beyond your wildest dreams")
+        if isDead:
```

Comment:
No where before this line is `isDead` set to True, so this code will never run (i.e., that's what PyCharm means by "dead code")

Reply:
This is still true. Notice there are no lines that say `isDead = True` before line 252. So, isDead will always be `False` here, meaning this code will never get run. 

Either you need to set `isDead` to True somewhere before this, or remove the condition, and run the code inside it in the right spot. 

As the designers, you two have to make that call.